setup.py

```diff
@@ -8,20 +8,31 @@
 from setuptools import setup, find_packages
 
 
-def get_version():
+# Make sources available using relative paths from this file's directory.
+#  dirname can return '', so use 'or' below to avoid chdir on empty
+#  while being succinct.
+os.chdir(os.path.dirname(__file__) or os.getcwd())
+
+
+def load_version():
     """Returns the version found in nilearn/version.py
 
     Importing nilearn is not an option because there may dependencies
     like nibabel which are not installed and setup.py is supposed to
     install them.
     """
-    locals_dict = {}
+    # load all vars into globals, otherwise
+    #   the later function call using global vars doesn't work.
     globals_dict = {}
-    with open(os.path.join('nilearn', 'version.py')) as fp:
-        exec(fp.read(), globals_dict, locals_dict)
+    execfile(os.path.join('nilearn', 'version.py'), globals_dict)
+
+    return globals_dict
+
+def is_installing():
+    return len(sys.argv) > 1 and sys.argv[1] == 'install'
```

Comment:
You need > 2 here. When using `python setup.py install`, sys.argv[0] is setup.py and sys.argv[1] is install.